book/src/tutorial/qt-gui/build-setup.md

```diff
@@ -0,0 +1,144 @@
+# Building a simple Qt application with Rust
+
+CXX-Qt provides helper libraries that make it easy for the Rust build system to link to Qt.
+
+The goal for now is to create a simple program that just launches an Qt/QML application.
```

Comment:
do we mention later that in theory this can be used for a widgets application too ? (eg for providing Rust models to a C++ widgets frontend)